packages/skatejs/src/with-initializer.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+export const withInitializer = (Base: Class<any> = HTMLElement): Class<any> =>
+  class extends Base {
+    static get observedAttributes() {
+      let observed = super.observedAttributes;
+      this.initializeCallback();
+      return observed;
+    }
+    static initializeCallback() {
```

Comment:
I've been thinking about names and I've undid the `Callback` suffix on all Skate specific stuff. I'd also like it to correspond to the `whenDefined()` functionality. Given that, I was going to suggest we call this `static defined()`. Thoughts?

Reply:
`defined` has meaning in the basic web component spec, right? Unless this meaning is the same as that meaning, I don't think they should be named similarly.

Reply:
Defined is specifically what we're trying to hook into without having to know the name and using `whenDefined()`.